### PR TITLE
Custom sidebar for pages, blog post and archive post

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

Created a single test case for :-
1. Sidebar position for Pages
2. Sidebar position for posts
3. Sidebar position for archive post
### How has this been tested?

Branch name :- custom-sidebar
Execution :- npm run test:e2e:interactive — tests/e2e/specs/customizer/sidebar/custom-sidebar.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/sidebar/custom-sidebar.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { setCustomize } from '../../../utils/set-customize';
+import { createURL,createNewPost,publishPost } from '@wordpress/e2e-test-utils';
+    describe( 'Position of the Sidebar for pages under the Customizer', () => {
+        it( 'Position of Sidebar for pages should apply correctly', async () => {
+            const pageSidebar = {
+            'single-page-sidebar-layout': 'left-sidebar',
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/custom-sidebar.test.js:6]: code indentation

File: tests/e2e/specs/customizer/sidebar/custom-sidebar.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    property: '',
+                } ).cssValueToBe( `` );  
+        });
+
+        it( 'Position of Sidebar for archive-posts should apply correctly', async () => {
+            const archivesidebar = {
+            'archive-post-sidebar-layout': 'left-sidebar',
+            };
+                await setCustomize( archivesidebar );
+                await page.goto( createURL( '2021/09' ), {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/custom-sidebar.test.js:44]: Enter valid URL

File: tests/e2e/specs/customizer/sidebar/custom-sidebar2.test.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                    property: '',
+                } ).cssValueToBe( `` );  
+        });
+
+        it( 'Position of Sidebar for archive-posts should apply correctly', async () => {
+            const archivesidebar = {
+            'archive-post-sidebar-layout': 'right-sidebar',
+            };
+                await setCustomize( archivesidebar );
+                await page.goto( createURL( '2021/09' ), {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/custom-sidebar2.test.js:44]: URL

File: tests/e2e/specs/customizer/sidebar/custom-sidebar3.test.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                    property: '',
+                } ).cssValueToBe( `` );  
+        });
+
+        it( 'Position of Sidebar for archive-posts should apply correctly', async () => {
+            const archivesidebar = {
+            'archive-post-sidebar-layout': 'no-sidebar',
+            };
+                await setCustomize( archivesidebar );
+                await page.goto( createURL( '2021/09' ), {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/custom-sidebar3.test.js:44]: URL

**Comment by bsfsnehal** [comment]
PR closed

**Pull request closed without merging**